packages/assets/test/assets.tests.ts

```diff
@@ -245,6 +245,36 @@ describe('Assets', () =>
         expect(bunny.baseTexture).toBe(null);
     });
 
+    it('should load TXT assets from data URL', async () =>
+    {
+        let txtDataURL = `
+        data:text/plain,Hello, world!
```

Comment:
Could you also add a test for base64-encoded data URIs:

```
data:text/plain;base64,SGVsbG8sIHdvcmxkIQ==
```

Reply:
Done! Tests for JSON & SVG are also updated.